mm2src/coins/eth/eth_balance_events.rs

```diff
@@ -102,7 +131,7 @@ async fn fetch_balance(
     let balance_as_big_decimal = u256_to_big_decimal(balance_as_u256, decimals).map_err(|e| BalanceFetchError {
         ticker: token_ticker.clone(),
         address: address.to_string(),
-        error: e.into(),
+        error: e.into_inner().into(),
```

Comment:
won't error tracing provided by MmError be lost? if you are calling `get_inner` to extract the error ?

Reply:
Indeed! and no idea why i did that xD
Let me try to revert it back, or at least log the traced error when it happens.

Reply:
[471c5db](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2172/commits/471c5dbca70563e10f589747b41802b3801114a8)